ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -267,9 +267,12 @@ interface {{ port_channel_interface }}
    ip ospf message-digest-key {{ ospf_message_digest_key }} {{ port_channel_interfaces[port_channel_interface].ospf_message_digest_keys[ospf_message_digest_key].hash_algorithm }} 7 {{ port_channel_interfaces[port_channel_interface].ospf_message_digest_keys[ospf_message_digest_key].key }}
 {%         endif %}
 {%     endfor %}
-{%     if port_channel_interfaces[port_channel_interface].pim.ipv4.sparse_mode is arista.avd.defined(true) %}
```

Comment:
I think you removed the true criteria from the wrong line, you probably meant to remove it on line 273? Ethernet and VLAN interfaces both look correct. This would fix your molecule output for port-channels as well, as you don't currently print out the priority command.